client/src/App.js

```diff
@@ -1,22 +1,29 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
 
 function App() {
+  const [data, setData] = React.useState(null);
+
+  /* Fetch API is a tool that manipulates HTTP and, at this moment is to perform requests by using a Get request. To do that
+  is to take in a promise which is a path of /api address that we want to fetch from and make our request. So basically to 
```

Comment:
@Hanna-N9 These are wonderfully thorough comments Hanna! I would maybe keep them a little bit more succinct so that it's easy for another dev to get context about the code with a quick note. Maybe something like `fetch allows for asyncronous http requests and returns a promise`